scripts/serve/serve-static.js

```diff
@@ -37,10 +37,10 @@ const staticApp = (site, index) => {
 const serve = (options) => {
   const {
     // Setting defaults here instead of in cli.
-    site = '.build/', port = 8080, host = '0.0.0.0', index,
+    site = './build', port = 8080, host = '0.0.0.0', index,
   } = options;
 
-  if (!dirExists(site)) {
+  if (!dirExistsWithContent(site)) {
     return Logger.warn(`Cannot serves content from ${site} because it does not exist.`, { context });
```

Comment:
We should update the message to add  ' or is empty'

Reply:
updated: https://github.com/cerner/terra-toolkit/pull/298/commits/9ff9152b60a139e62cc5c4e31afaddf077668ccf#diff-2d0e39e2b5431dc8d9df655f5fc376fbR44